Userland/Libraries/LibVideo/VP9/Decoder.cpp

```diff
@@ -807,14 +806,6 @@ DecoderErrorOr<void> Decoder::predict_inter_block(u8 plane, u8 ref_list, u32 x,
         return block_buffer[row * width + column];
     };
 
-    /*for (auto row = 0u; row < height; row++) {
```

Comment:
see other comment for ref but this should be deleted ~5 commits earlier so that it's never even added :) 

(commenting on `LibVideo: Remove unnecessary dbgln calls`).